### PR TITLE
docs: add Nimble setup to Wii U install doc

### PR DESCRIPTION
**Pull request by LiamMGit**
(please let me know if this PR didn't work properly, i really hope it didn't screw up D:)
This just adds a basic message saying what to do after your PNID is setup, also adds the link to the "Inkay" section.
Feel free to ask me for any changes!

### CODE REVIEW

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:59]:
```suggestion
Nimble is a Aroma plugin that allows your console to recive SpotPass data again. Download the latest version from [Nimble's latest release](https://github.com/PretendoNetwork/Nimble/releases/latest) on GitHub and download the `30_nimble.rpx` file from the latest release.
```

Also this is a runon sentence. Can probably be paced better

Also Nimble is used for the policy list, not BOSS in general? BOSS content in games in general should still be working even without Nimble. @ashquarky Can you confirm this? Did Nintendo's shutdown somehow break BOSS content downloads in games?

Also CC @DaniElectra and @shutterbug2000 for input as well

Maybe we should look into trying to bundle Nimble into Inkay in this case? Nimble being separate made sense before when WWP was the only thing it really helped with, and not everyone cared about that, but if it's required for BOSS in general now that's a bigger issue

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:92]: You duplicated a lot of this information in various places, I feel like this could be structured differently?

**Review by jonbarrow — CHANGES_REQUESTED** [review]

File: docs/en_US/install/wiiu.md
Comment (LiamM-MTCBs) [reply docs/en_US/install/wiiu.md:92]: Thanks for the feedback, I'll try and fix some stuff up.

**Comment by LiamM-MTCBs** [comment]
Tried my best to change it up a little, changed some structuring and changed the explanation for what Nimble is. Let me know if you want any more changes!

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:59]: My previous comment on this may have been confusing. Games like Splatoon download their content through BOSS (SpotPass). Inkay already contains patches for BOSS, so those games should already be covered.

Nimble specifically patches the policy list. This is not on it's own server, it is also part of the BOSS server. But the reason it's in its own patch is because the console downloads the policy list too early in the boot process for Inkay to handle it. This is then used by tasks like Wara Wara Plaza to download content at system boot.

This is why I tagged @ashquarky, @DaniElectra and @shutterbug2000 for opinions before with regards to merging the patches and to confirm if the Nintendo shut down affected things on our end in unexpected ways. Please do not mark this as resolved until that has been discussed

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:109]: This would probably be better placed before PNID setup. It feels odd putting this additional patch guide in the middle of the account setup steps, especially since it doesn't really have much to do with the account

**Review by jonbarrow — CHANGES_REQUESTED** [review]
Please use more detailed commit messages. Commit messages like `Update wiiu.md` do not tell anyone looking at the commit history *what* changes were made

File: docs/en_US/install/wiiu.md
Comment (LiamM-MTCBs) [reply docs/en_US/install/wiiu.md:109]: Thanks for the advice, will change up a few things soon when I have time. Very much appreciated.

File: docs/en_US/install/wiiu.md
Comment (LiamMGit) [reply docs/en_US/install/wiiu.md]: Alright, let me know on any updates you get.

File: docs/en_US/install/wiiu.md
Comment (LiamMGit) [reply docs/en_US/install/wiiu.md]: I changed it so its a part of the installation process do you think that's good or should I still make it its own separate thing?

File: docs/en_US/install/wiiu.md
Comment (ashquarky) [reply docs/en_US/install/wiiu.md]: I'm not aware of any shutdown-related breakage other than the policylist being all suspended now, which Nimble fixes. I was under the impression some games (like Splatoon) actually use the policylist though? Either way we need Nimble now.

Nimble needs to be a separate file for boring technical reasons (Aroma plugins run too *late* in the boot process) but we could certainly offer a zip with everything already arranged in it, like aroma.foryour.cafe does.

File: docs/en_US/install/wiiu.md
Comment (ashquarky) [inline docs/en_US/install/wiiu.md:84]: I don't blame you for not knowing this, but the bleeding-edge version of Inkay has actually changed these texts to be less technical and confusing.
It now says "Other settings" and "Reset Wara Wara Plaza". That's only in bleeding-edge though, not latest - @jonbarrow thoughts?

https://github.com/PretendoNetwork/Inkay/commit/216942cb64400a2140cbd8ed9c0bee2e8e7a2495#diff-6b2f0a449fdefd8930e23ef0dcd752beec69242e1303d77653f047c5e0766385R133

**Review by ashquarky — COMMENTED** [review]
We really need to get that installer app happening already, I think. Our complexity is getting pretty high..
Good docs changes though

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [reply docs/en_US/install/wiiu.md]: > I'm not aware of any shutdown-related breakage other than the policylist being all suspended now, which Nimble fixes. I was under the impression some games (like Splatoon) actually use the policylist though? Either way we need Nimble now.
> 
> 
> 
> Nimble needs to be a separate file for boring technical reasons (Aroma plugins run too *late* in the boot process) but we could certainly offer a zip with everything already arranged in it, like aroma.foryour.cafe does.

I know it needs to be separate, but any chance they could be built at the same time from the same repo? Or would trying to merge them be too much. It's been a while since I've looked at the Nimble repo

File: docs/en_US/install/wiiu.md
Comment (LiamM-ALT) [reply docs/en_US/install/wiiu.md:84]: Should I make it temporarily like 'Reset Wara Wara Plaza' OR 'Unregister Wara Wara Plaza BOSS tasks', etc?

**Comment by jonbarrow** [comment]
> We really need to get that installer app happening already, I think. Our complexity is getting pretty high..
> 
> Good docs changes though

I agree. There's quite a lot now, and it's getting harder to manage it all

File: docs/en_US/install/wiiu.md
Comment (LiamM-ALT) [reply docs/en_US/install/wiiu.md:84]: > Should I make it temporarily like 'Reset Wara Wara Plaza' OR 'Unregister Wara Wara Plaza BOSS tasks', etc?

Ah crap, on my alt, whoops!

File: docs/en_US/install/wiiu.md
Comment (ashquarky) [reply docs/en_US/install/wiiu.md]: They could be folders in the same repo I suppose, there's not really any overlap code-wise since Inkay is WUPS and Nimble is basically a normal homebrew app

I assume you want this so that we have one GitHub release?

File: docs/en_US/install/wiiu.md
Comment (ashquarky) [reply docs/en_US/install/wiiu.md:84]: Ehhh, I guess leave it - we only just pushed Inkay v2.4 out so it'll be a bit before we do another release (unless something comes up)

Comment (LiamMGit) [reply docs/en_US/install/wiiu.md]: > Ehhh, I guess leave it - we only just pushed Inkay v2.4 out so it'll be a bit before we do another release (unless something comes up)

Just saw this tight after i pushed out the commit lol, will revert it

Comment (LiamMGit) [reply docs/en_US/install/wiiu.md]: Reverted.

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [reply docs/en_US/install/wiiu.md]: > They could be folders in the same repo I suppose, there's not really any overlap code-wise since Inkay is WUPS and Nimble is basically a normal homebrew app
> 
> 
> 
> I assume you want this so that we have one GitHub release?

Yeah that would be the goal. But if it's not feasible that's 100% fine. An installer would solve these issues too

**Comment by ashquarky** [comment]
I think this is fine to merge unless Jon has any changes - the situation with Nimble isn't great; but the docs need to reflect the current state of things - which these do.

**Comment by has207** [comment]
Any chance of this getting committed soon? Not an expert on this codebase but it looks like a simple docs change that's actually pretty important for anyone installing on homebrew consoles.

**Comment by jonbarrow** [comment]
> it looks like a simple docs change

It is. But we have many dozens of issues and PRs across all of our repositories now, and it takes time to go through them all and do proper reviews on top of our other general server work and research. Even "simple docs changes" like these need proper reviews as there's still standards and rules to be followed (which you can see by the reviews here)

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:59]: Corrects what Nimbus is (it's not a plugin) and explains what it's used for a bit better

```suggestion
Nimble is an Aroma setup module that patches the request for the system `PolicyList`. This is used to enable background tasks for features such as Wara Wara Plaza. This is separate from the main Inkay patches as the `PolicyList` is downloaded earlier in the system boot process than Inkay has control over.

Navigate to [the latest release](https://github.com/PretendoNetwork/Nimble/releases/latest) on GitHub and download the  `30_nimble.rpx` file.
```

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:68]: Makes this match the wording of the above steps

```suggestion
Place the downloaded `30_nimble.rpx` file on your SD card at `sd:/wiiu/environments/aroma/modules/setup`. It's normal for other Aroma setup modules to be in this folder too.
```

File: docs/en_US/install/wiiu.md
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:73]: This is said already on the very next line

```suggestion
```

**Review by jonbarrow — CHANGES_REQUESTED** [review]
This should be it. Just some grammar fixes and better explaining some parts really

**Pull request merged**
